gkeys/gkeys/mail.py

```diff
@@ -0,0 +1,49 @@
+from __future__ import print_function
+
+import os
+import smtplib
+import sys
+
+
+from email.mime.text import MIMEText
+from snakeoil.demandload import demandload
+
+if sys.version_info[0] >= 3:
+    py_input = input
+    _unicode = str
+else:
+    py_input = raw_input
+    _unicode = unicode
+
+demandload(
+    "gkeys.base:Args",
+    "json:load",
+)
+
+class Emailer(object):
+    '''Send an email reminder about the status of the user's GPG key'''
+
+    def __init__(self, login, logger):
+        self.logger = logger
+        self.email_from = _unicode(login['login_email'])
+        login_passwd = login['passwd']
+        server = login['server']
+        self.mail = smtplib.SMTP(server, 587)
+        self.mail.ehlo()
+        self.mail.starttls()
+        self.mail.login(self.email_from, login_passwd)
+        self.logger.debug(_unicode("Login successfull"))
+
+    def send_email(self, uid, message):
+        self.logger.debug(_unicode('Sending email with message %s') % _unicode(message))
+        subject = "Expiring Key"
+        email_to = _unicode("justthisthing@gmail.com") #uid
```

Comment:
I know this is hard-coded to you for your testing, but it could be done a little better so that it is clear it is only for debug testing.  

1) you could have the logger add a debug message stating the uid target. Add a comment line stating this is for a debug testing override.

2) you could add a --debug-target option to the spec-check action which overrides all email target uid's only at this point.